CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,50 @@
+# Contributor Code of Conduct
+
+As contributors and maintainers of this project, and in the interest of
+fostering an open and welcoming community, we pledge to respect all people who
+contribute through reporting issues, posting feature requests, updating
+documentation, submitting pull requests or patches, and other activities.
+
+We are committed to making participation in this project a harassment-free
+experience for everyone, regardless of level of experience, gender, gender
+identity and expression, sexual orientation, disability, personal appearance,
+body size, race, ethnicity, age, religion, or nationality.
+
+Examples of unacceptable behavior by participants include:
+
+* The use of sexualized language or imagery
+* Personal attacks
+* Trolling or insulting/derogatory comments
+* Public or private harassment
+* Publishing other's private information, such as physical or electronic
+  addresses, without explicit permission
+* Other unethical or unprofessional conduct
```

Comment:
I think that trying to define all of these things specifically will take us down a rabbit hole. You could ask a similar question for just about every bullet point in this list. Ultimately it is up to the community to make the subjective judgment calls on any conduct.

An example of unprofessional conduct could be if a person used vulgar language when communicating on a pull request. But there are many other possibilities of unprofessional conduct. Listing them specifically should not be the goal of the code.